n2y/plugins/jinjarenderpage.py

```diff
@@ -142,19 +177,25 @@ def render_from_string(source, context=None, environment=None):
 class JinjaRenderPage(Page):
     def __init__(self, client, notion_data):
         super().__init__(client, notion_data)
-        self.client.plugin_data['jinjarenderpage'] = {
-            'environment': _create_jinja_environment(),
```

Comment:
Add a comment to these lines https://github.com/innolitics/n2y/blob/main/n2y/notion.py#L185-L192

explaining the problem that lead to us checking the page plugin class type. Then note that now we typically use the same plugins for all pages, and thus the check is likely no longer necessary.

I'd also add a logger warning call noting that the existing page class got replaced.